src/containers/enrollment/EnrollmentLink.js

```diff
@@ -0,0 +1,42 @@
+import { faCopy } from '@fortawesome/pro-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// @flow
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardSegment,
+} from 'lattice-ui-kit';
+
+import copyToClipboard from '../../utils/copyToClipboard';
+
+const EnrollmentLink = () => {
+  const enrollmentLinkQueryParams = window.location.hash.split('?')[1];
+  const enrollmentLink = `http://openlattice.com/login?${enrollmentLinkQueryParams}`;
+  return (
+    <>
+      <Card>
+        <CardHeader mode="danger">
+          Page Not Found
+        </CardHeader>
+        <CardSegment>
+          You have tried to open a mobile link in a web browser. Please install the Chronicle mobile application
+          and open the link on a mobile device.
```

Comment:
I think the wording needs to change a bit. I think it would be clearer to refer to it as the "enrollment link". Additionally, the behavior is the same whether you open it on mobile/desktop, i.e. you still get a 404. How about something like this ...

```
You have tried to open the Chronicle enrollment link in a web browser. Please install the Chronicle mobile application and open the enrollment link with it.
```

I don't like the "with it" at the end there, just couldn't think of a better suggestion.

Reply:
how about ... `and open the link your device.`

Reply:
Oh good point. Thanks! I changed it to this—does this make sense?

`You have tried to open the Chronicle enrollment link in a web browser. Please install the
          Chronicle mobile application and try opening the link again.`